rust/processor/src/db/common/models/ans_models/parquet_ans_lookup_v2.rs

```diff
@@ -0,0 +1,30 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+// This is required because a diesel macro makes clippy sad
+#![allow(clippy::extra_unused_lifetimes)]
+#![allow(clippy::unused_unit)]
+
+use field_count::FieldCount;
+use serde::{Deserialize, Serialize};
+
+type Domain = String;
+type Subdomain = String;
+type TokenStandardType = String;
+type RegisteredAddress = String;
+// PK of current_ans_lookup_v2
+type CurrentAnsLookupV2PK = (Domain, Subdomain, TokenStandardType);
+// PK of current_ans_primary_name
+type CurrentAnsPrimaryNameV2PK = (RegisteredAddress, TokenStandardType);
+
+#[derive(Clone, Default, Debug, Deserialize, FieldCount, Serialize)]
+pub struct AnsPrimaryNameV2 {
+    pub txn_version: i64,
+    pub write_set_change_index: i64,
+    pub registered_address: String,
+    pub token_standard: String,
+    pub domain: Option<String>,
+    pub subdomain: Option<String>,
+    pub token_name: Option<String>,
+    pub is_deleted: bool,
```

Comment:
current table also includes `expiration_timestamp` field

missing `block_timestamp`

Reply:
do you need both ans_primary_name_v2 and current_ans_primary_name_v2 in parquet or just current one?

Reply:
@rtso deos current_ans_primary_name_v2 only contains curren one as well or does it include v1?

Reply:
confirmed with @rtso that `current_ans_primary_name_v2` includes v1

Reply:
for posterity, current_ans_primary_name_v2 contains v1